src/Plugin/RedirectPlugin.php

```diff
@@ -244,22 +249,6 @@ private function createUri(ResponseInterface $redirectResponse, RequestInterface
             $uri = $uri->withPort($parsedLocation['port']);
```

Comment:
i am actually not sure about this one either. if i was on http://localhost:8080/foo and i get redirect a header https://php-http.org/bar i would not expect to stay on port 8080... the only case where the port should be kept is when there is also no host in the redirection... wdyt?

Reply:
I agree. Will change